### PR TITLE
Add v1.2.1 to v1.2.2 upgrade page

### PR DESCRIPTION
**Pull request by bk201**
Add v1.2.2 upgrade matrix.

### CODE REVIEW

**Review by starbops — APPROVED** [review]
LGTM, thanks!

File: docs/upgrade/v1-2-1-to-v1-2-2.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/upgrade/v1-2-1-to-v1-2-2"/>
+</head>
+
+## General information
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```
Comment (jillian-maroket) [inline docs/upgrade/v1-2-1-to-v1-2-2.md:15]:
```suggestion
An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```

File: versioned_docs/version-v1.2/upgrade/v1-2-1-to-v1-2-2.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/upgrade/v1-2-1-to-v1-2-2"/>
+</head>
+
+## General information
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.2/upgrade/v1-2-1-to-v1-2-2.md:15]:
```suggestion
An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```

File: versioned_docs/version-v1.3/upgrade/v1-2-1-to-v1-2-2.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/upgrade/v1-2-1-to-v1-2-2"/>
+</head>
+
+## General information
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will show an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
+
+For the air-gap env upgrade, please refer to [prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.3/upgrade/v1-2-1-to-v1-2-2.md:15]:
```suggestion
An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
```

File: docs/upgrade/v1-2-1-to-v1-2-2.md (hunk 1 above)
Comment (jillian-maroket) [reply docs/upgrade/v1-2-1-to-v1-2-2.md:15]: This wording is from [Upgrade from v1.1.1/v1.1.2 to v1.1.3](https://docs.harvesterhci.io/v1.4/upgrade/v1-1-1-to-v1-1-3#general-information).

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

File: docs/upgrade/v1-2-1-to-v1-2-2.md (hunk 1 above)
Comment (bk201) [reply docs/upgrade/v1-2-1-to-v1-2-2.md]: Fixed. Thanks for adding comments to all versions!

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**